user/useradm.go

```diff
@@ -423,6 +439,20 @@ func (ua *UserAdm) Verify(ctx context.Context, token *jwt.Token) error {
 		return errors.Wrap(err, "useradm: failed to get token")
 	}
 
+	// in case the token is a personal access token, update last used timestam
+	// to not overload the database with writes to tokens collection, we do not
+	// update the timestamp every time, but instead we wait some configurable
+	// amount of time between updates
+	if dbToken.TokenName != nil && ua.config.TokenLastUsedUpdateFreqMinutes > 0 {
+		t := time.Now().Add(
+			(-time.Minute * time.Duration(ua.config.TokenLastUsedUpdateFreqMinutes)))
+		if dbToken.LastUsed == nil || dbToken.LastUsed.Before(t) {
+			if err := ua.db.UpdateTokenLastUsed(ctx, token.ID); err != nil {
+				return err
+			}
+		}
+	}
+
```

Comment:
Line 477: Shouldn't all tokens (both personal access tokens and short-lived ones) be removed when user is deleted?

Reply:
probably should; fixed with new commit

Reply:
looks good now!